manifests/RProject/R/4.0.0.yaml

```diff
@@ -0,0 +1,12 @@
+Id: RProject.R
+Publisher: RProject
+Name: RProject R
```

Comment:
I think the name of the application would just be R... I don't usually hear people say they look for the Rproject R. 
Usually it is just R for the one from CRAN, and other name like MRO for the distribution provided by microsoft. 

Seems a bit strange this naming. Curious of what other think... 🤔

Reply:
I agreed. I'll change the ```name``` from ```RProject R``` to ```R```.

Reply:
In fact reading the [spectification](https://docs.microsoft.com/en-us/windows/package-manager/package/manifest?tabs=minschema%2Cbetter), maybe you can keep this name and use the shorter `AppMoniker` field.
In any case, we should add 
```yaml
AppMoniker: R
```

Reply:
I think changing ```name``` from ```RProject R``` to just ```R``` might be a better idea, while also including ```AppMoniker: R```. I'll wait for some feedback on this before I update the code.